src/main/java/core/basesyntax/ConsoleHandler.java

```diff
@@ -0,0 +1,174 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.stream.Stream;
+
+public class ConsoleHandler {
+
+    private static final Map<String, String> commandInfo = Map.of("create",
+            "create [path] [file-name]\nCreate file by specified path.\n", "read",
+            "read [path] [file-name]\nRead a file by specified path to the terminal.\n",
+            "info", "info [path] [file-name]\nGet short info about specified file.\n",
+            "help", "help [command]\n"
+                    + "Get info about all available commands or about a specified one.\n",
+            "exit", "exit\nSay bye-bye :-)\n");
+
+    public void handleInput() throws IOException {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String input = scanner.nextLine();
+            String[] args = input.split(" ");
+            String pathOrCommandName = null;
```

Comment:
variable should never be used to have two different meanings. just use two variables.

Reply:
I had to write commandName as it was also used by help method. now it doesn't, so I just left pathName, thank you